static/js/theme-map/Util/helpers.js

```diff
@@ -55,8 +55,25 @@ const isViewableWithinContainer = (targetEl, containerEl) => {
   return isScrolledIntoView;
 };
 
+
+/**
+ * Normalize lng to the range [-180, 180]. If you give -181, for
+ * example, we wrap back to 179. If lng is 180, we return 180. Otherwise, we
+ * prefer to return -180 over 180 when wrapping, as they are the same coordinate.
+ *
+ * @param {Number} lng The longitude
+ * @returns {Number} The normalized longitude
+ */
+const getNormalizedLongitude = (lng) => {
+  if (lng === 180) {
+    return lng;
+  }
+  return ((lng + 180) % 360 + 360) % 360 - 180;
```

Comment:
Is there any way that you could add explanatory variables to help describe the how this formula works? Just looking at it, it's not clear to me how this formula was derived. If not, could you add a comment which shows your reasoning behind the formula?

Reply:
agree on the variables! just to double check I think the process is something like 
1. add 180 to convert to 0 to 360
2. mod 360 to normalize
3. add 360 take mod 360 AGAIN to make the mod positive (in javascript -4 % 10 is -4 not 6, but here we want 6)
4. subtract 180 to convert back to -180 to 180